polkadot/roadmap/implementers-guide/src/protocol-validator-disabling.md

```diff
@@ -0,0 +1,282 @@
+
+A few options for the duration of disablement were considered:
+- 1 epoch (4h in Polkadot)
+- 1 era (24h in Polkadot)
+- 2-26 eras
+- 27 eras
+
+1 epoch is a short period and between a few epochs the validator set might be exactly the same. It is also very difficult to fix any local node issues for honest validator in such a short time so the chance for a repeated offense is high.
+
+1 era gives a bit more time to fix any minor issues. Additionally, it guarantees a validator set change at so many of the currently disabled validator might no longer be present anyway. ([**Point 4.**](#system-overview))
+
+Higher values could be considered and the main arguments for those are based around the fact that it reduces the number of repeated attacks that will be allowed before the slash execution. Generally 1 attack per era for 27 eras resulting in 27 attacks at most should not compromise oru safety assumptions. Although this direction could be further explored and might be parametrized for governance to decide. 
+
+</br></br></br>
+
+# Economic consequences of Disablement
+
+Disablement is generally a form of punishment and that will be reflected in the rewards at the end of an era. A disabled validator will not receive any rewards for backing or block authoring. which will reduce it's profits.
+
+That means that the opportunity cost of being disabled is a punishment by itself and thus it can be used for some cases where a minor punishment is needed. Current implementation was using 0% slashes to mark nodes for chilling and similar approach of 0% slashes can be used to mark validators for disablement. ([**Point 1.**](#system-overview))
+0% slashes could for instance be used to punish approval checkers voting invalid on valid candidates.
+
+Anything higher than 0% will of course also lead to a disablement.
+
+> **Notes:** \
+> Alternative designs incorporating disabling proportional to offenses were explored but they were deemed too complex and not worth the effort. Main issue with those is that proportional disabling would cause back and forth between disabled and enabled which complicated tracking the state of disabled validators and messes with optimistic node optimizations. Main benefits were that minor slashes will be barely disabled which has nice properties against sacrifice attacks.
+
+</br></br></br>
+
+# Redundancy
+
+Some systems can be greatly simplified our outright removed thanks to the above changes. This leads to reduced complexity around the systems that were hard to reason about and were sources of multiple bugs.
+
+## Automatic Chilling
+
+Chilling is process of a validator dropping theirs intent to validate. This removes them from the upcoming NPoS solutions and effectively pushes them out of the validator set as quickly as of the next era (or 2 era in case of late offenses). All nominators of that validator were also getting unsubscribed from that validator. Validator could re-register their intent to validate at any time.
```

Comment:
>  All nominators of that validator were also getting unsubscribed from that validator. Validator could re-register their intent to validate at any time.

This was initially the case but it was removed to avoid punishing validators slashes that were reverted due to bugs, etc, and avoid that the wrongly punished validators needed to bootstrap all over again again.

Reply:
Expanded this point. 

Mentioning that it was the case, there were issues and it was consequently removed.
Adding that after it was removed we have the issue of offenders simply re-registering validation interest and continuing to commit offences. No good middle-ground between the two.